src/libraries/System.Security.Cryptography.Encoding/tests/Base64TransformsTests.cs

```diff
@@ -115,6 +115,38 @@ public static void ValidateFromBase64CryptoStream(string data, string encoding)
             }
         }
 
+        [Theory]
+        [InlineData(100)]
+        [InlineData(101)]
+        [InlineData(102)]
+        [InlineData(103)]
+        public static void RoundtripCryptoStream(int length)
+        {
+            byte[] expected = RandomNumberGenerator.GetBytes(length);
+            var ms = new MemoryStream();
```

Comment:
Maybe dispose the MemoryStream at the end or set the last ``leaveOpen`` to false?

Reply:
I rarely dispose MemoryStreams, especially in tests, as MemoryStream.Dispose is a nop.